index.bs

```diff
@@ -2139,7 +2139,7 @@ Issue: This needs to be generalized to work with realms too
         params: SubscribeParameters
       }
 
-      SessionSubscribeParameters = {
+      SubscribeParameters = {
```

Comment:
I wonder how it would look like when several modules would have a `subscribe` command that need parameters. Maybe then it would make sense to prefix with the module name?

`script.callFunction` has btw the same issue.

Reply:
Agree. Module prefix is a good idea. Renamed to 

Issue with `script.callFunction`, do you mean `ArgumentValue`, which is too vogue and can interfere with other modules?